rules/opentitan_test.bzl

```
@@ -5,8 +5,9 @@
 load("@//rules:opentitan.bzl", "opentitan_flash_binary", "opentitan_rom_binary")
 load("@bazel_skylib//lib:shell.bzl", "shell")
 
-_EXIT_SUCCESS = r"PASS.*\n"
-_EXIT_FAILURE = r"(FAIL|FAULT).*\n"
+DEFAULT_TEST_SUCCESS_MSG = r"PASS.*\n"
+DEFAULT_TEST_FAILURE_MSG = r"(FAIL|FAULT).*\n"
+ROM_BOOT_FAILURE_MSG = "BFV:[0-9a-f]{8}"
```

I think we should call these something like `OTTF_{SUCCESS,FAILURE}_MSG`. We can then define the default success msg as equal to `OTTF_SUCCESS_MSG` and the default failure msg as the disjunction of both failure messages, i.e.

```suggestion
OTTF_SUCCESS_MSG = r"PASS.*\n"
OTTF_FAILURE_MSG = r"(FAIL|FAULT).*\n"
ROM_BOOT_FAILURE_MSG = "BFV:[0-9a-f]{8}"

DEFAULT_TEST_SUCCESS_MSG = OTTF_SUCCESS_MSG
DEFAULT_TEST_FAILURE_MSG = "{}|{}".format(OTTF_FAILURE_MSG, ROM_BOOT_FAILURE_MSG)
```

This way, the default message would catch both boot failures and test failures.

You might also want to add a short note that says sthg like "these are defined for positive test cases and should be flipped for negative test cases".

---

great idea! fixed.